core/io/resource_saver.h

```diff
@@ -80,7 +81,7 @@ class ResourceSaver {
 		FLAG_REPLACE_SUBRESOURCE_PATHS = 64,
 	};
 
-	static Error save(const String &p_path, const RES &p_resource, uint32_t p_flags = 0);
+	static Error save(const String &p_path, const RES &p_resource, uint32_t p_flags = static_cast<uint32_t>(FLAG_NONE));
```

Comment:
We prefer C-style casts:
```suggestion
	static Error save(const String &p_path, const RES &p_resource, uint32_t p_flags = (uint32_t)FLAG_NONE);
```